tests/unit/amo/reducers/test_users.js

```diff
@@ -0,0 +1,206 @@
+import reducer, {
+  FETCH_CURRENT_USER_ACCOUNT,
+  FETCH_USER_ACCOUNT,
+  LOAD_USER_ACCOUNT,
+  getCurrentUser,
+  getDisplayName,
+  fetchCurrentUserAccount,
+  fetchUserAccount,
+  hasPermission,
+  initialState,
+  loadCurrentUserAccount,
+  loadUserAccount,
+  logOutUser,
+} from 'amo/reducers/users';
+import {
+  ALL_SUPER_POWERS,
+  ADMIN_TOOLS_VIEW,
+  STATS_VIEW,
+  THEMES_REVIEW,
+} from 'core/constants';
+import {
+  dispatchClientMetadata,
+  dispatchSignInActions,
+} from 'tests/unit/amo/helpers';
+import {
+  createStubErrorHandler,
+  createUserAccountResponse,
+} from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('initializes properly', () => {
+      const { byID, byUsername } = reducer(undefined, { type: 'NONE' });
+      expect(byID).toEqual({});
+      expect(byUsername).toEqual({});
+    });
+
+    it('ignores unrelated actions', () => {
+      const state = reducer(undefined, loadUserAccount({
+        user: createUserAccountResponse({ id: 12345, username: 'john' }),
+      }));
+      const newState = reducer(state, { type: 'UNRELATED' });
+      expect(newState).toEqual(state);
+    });
+
+    it('requires errorHandlerId for fetchCurrentUserAccount', () => {
+      expect(() => {
+        fetchCurrentUserAccount({});
+      }).toThrowError('errorHandlerId is required');
+    });
+
+    it('fetchCurrentUserAccount returns action', () => {
```

Comment:
This test doesn't seem useful to me. You will already have coverage of this when testing interaction with the reducer.